cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/messages/MappedIterator.java

```diff
@@ -23,26 +23,33 @@ public abstract class MappedIterator<S, T> implements Iterator<T>
 {
 	private final int limit;
 	private final AtomicInteger returned;
-	private final Iterator<T> targetIterator;
+	private final Iterator<S> sourceIterator;
+	private Iterator<T> targetIterator;
```

Comment:
Previosly targetIterator was final because it was instantiated in constructor. But it is possible that child class will need something which is only available after object instance creation. With previous approach it will be impossible to use variables which are available only after object instance creation: super constructor have to be called before child constructor